source/deprecations/v1.x.html.md

```diff
@@ -1172,6 +1230,12 @@ Object.freeze(c);
 c.swapNames();  // EXCEPTION
 ```
 
+#### Ember.EnumerableUtils
+
+`EnumerableUtils` has been deprecated in favor of native implementations. You can consult the
+[JavaScript Array documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array)
+to find suitable replacements for `EnumerableUtils` functionality.
+
```

Comment:
We might want to suggest `underscore` or `lodash` if the functionality a user wants isn't available in the native implementations.